search/src/Components/dataset/dataset_list.js

```diff
@@ -70,8 +71,9 @@ export default function DatasetList(props) {
          })
   }
 
-  const datalist_table = (table_list, command) => (
+  return (
     <div className={classes.root}>
+      <h2>{props.title}</h2>
```

Comment:
okay. But since this is not set in the non-staff case, should we make the h2 conditional?